pages/linux/aspell.md

```diff
@@ -0,0 +1,23 @@
+# aspell
+
+> Interactive spell checker.
+
+- Spell check a single file:
+
+`aspell check file`
+
+- List mispelled words from standard input:
+
+`cat file | aspell list`
+
+- Show available dictionary files:
```

Comment:
Perhaps we could change this to `Show available dictionary languages` instead? I get an output something like this on my machine:

```
// .....
en_GB
en_GB-ise
en_GB-ise-w_accents
en_GB-ise-wo_accents
en_GB-ize
en_GB-ize-w_accents
en_GB-ize-wo_accents
en_GB-variant_0
en_GB-variant_1
en_GB-w_accents
en_GB-wo_accents
en_US
// .....
```

This might help the next command to make more sense.

Reply:
done